test/test_encoding.py

```diff
@@ -0,0 +1,743 @@
+#!/usr/bin/python3 -u
+"""
+SPDX-License-Identifier: BSD-2
+"""
+import unittest
+
+from tpm2_pytss import *
+from tpm2_pytss.encoding import (
+    base_encdec,
+    toolslike_encdec,
+    json_encdec,
```

Comment:
I'm assuming this matches the json spec?
  - https://trustedcomputinggroup.org/resource/tcg-tss-json/

Reply:
That's the idea